src/main/java/net/dv8tion/jda/internal/requests/Route.java

```diff
@@ -517,4 +515,19 @@ public String getRatelimitRoute()
             return "channels/%s/messages/{message_id}/delete"; //the additional "/delete" forces a new bucket
         }
     }
+
+    // This endpoint shares the rate-limit bucket with REMOVE_ALL_REACTIONS
+    private static class ReactionRoute extends Route
+    {
+        private ReactionRoute(Method method)
+        {
+            super(method, new RateLimit(1, 250), "channels/{channel_id}/messages/{message_id}/reactions/{reaction_code}/{user_id}", "channel_id");
```

Comment:
Does the `REMOVE_REACTION` endpoint share the same `1 reaction / 250ms` limit that `ADD_REACTION` does?

Additionally, if that is the case, does `REMOVE_ALL_REACTIONS` also share that limit?

How does this actually effect the system now that they have conflicting route bucket keys?

Reply:
My tests indicate that all these routes have the same shared rate limit. This also includes the reset time but I'll check again.

Reply:
According to my tests this did not encounter 429 once in 10 tries.
```
03:25:40.988 JDA-ThreadPool 9  RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:41.401 JDA-ThreadPool 12 RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:41.823 JDA-ThreadPool 12 RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:42.248 JDA-ThreadPool 12 RateLimiter DEBUG  Backing off 249 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:42.652 JDA-ThreadPool 2  RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:43.068 JDA-ThreadPool 12 RateLimiter DEBUG  Backing off 249 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:43.482 JDA-ThreadPool 1  RateLimiter DEBUG  Backing off 249 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:43.903 JDA-ThreadPool 6  RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:44.312 JDA-ThreadPool 2  RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions
03:25:44.729 JDA-ThreadPool 2  RateLimiter DEBUG  Backing off 250 milliseconds on route /channels/163772719836430337/messages/{message_id}/reactions```